roles/common/tasks/k8s.yml

```diff
@@ -231,3 +231,35 @@
       kubectl --context $ctx create secret generic {{ item.name }} --from-literal=token={{ item.token }} --dry-run=client -o yaml | kubectl --context $ctx apply -f -
     done
   loop: "{{ kci_tokens.tokens }}"
+
+- name: Create /home/buildslave/bin
+  tags:
+    - k8s-scripts
+    - never
+  file:
+    path: "/home/buildslave/bin"
+    state: directory
+    owner: buildslave
+    group: buildslave
+    mode: 0755
+    
+- name: Add k8s helper shell scripts
+  tags:
+    - k8s-scripts
+    - never
+  copy:
+    src: k8s/cleanup.sh
+    dest: "/home/buildslave/bin/k8s-cleanup.sh"
+    owner: buildslave
+    group: buildslave
+    mode: 0755
+
+- name: Daily cron for k8s garbage collection
```

Comment:
Can this be easily turned into a systemd timer instead of cron?  Or is Ansible going to create a systemd timer anyway with `ansible.builtin.cron`?

Reply:
I assumed it would create a systemd timer, but in fact it creats a cron entry for the buildslave user.  I'll need to read ansible docs to see if there's an easy way.